src/sql/azdata.proposed.d.ts

```diff
@@ -767,6 +767,20 @@ declare module 'azdata' {
 		delete?: boolean;
 	}
 
+	export enum CardType {
+		Image = 'Image'
+	}
+
+	export interface CardImage {
+		path: vscode.Uri,
+		size?: string, // auto|length|cover|contain|initial|inherit
+		position?: string // (left | right | center) (top | center | bottom) | x% y% | xpos ypos | initial | inherit
+	}
+
+	export interface CardProperties extends ComponentWithIconProperties {
```

Comment:
We already have icon properties on the card so I think it'd be less confusing to try and re-use those. I also think that for types like these we should try to avoid stuffing everything into a single properties class and instead have superclasses (`ImageCardProperties` for example) that contain the properties used by an image card. Makes it much easier to work with in general. 

@alanrenmsft thoughts here? Unfortunately this card stuff is already kind of a mess so there's only so much we can do....

Reply:
Ok. I thought that making it work as a background image would be stretching out its purpose too far, but if it makes sense to you, that works for me. I'll use the icon to load the thumbnail into the card.

Reply:
It's not my favorite and if we were starting this from scratch I'd look into a better name for the properties - but for this I think it's better to reduce the number of properties we have that are basically doing the same thing.

Reply:
Makes sense. I changed card image to use the icon. Let me know if the implementation makes sense to you.